boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -99,7 +103,7 @@ op-generate:
 .PHONY: generate
 generate: op-generate go-generate
 
-# rporresm: Force GOOS=linux as we may want to build containers in darwin
+# upstream-diff: Force GOOS=linux as we may want to build containers in darwin
```

Comment:
We're still in the process of figuring out how to make local targets run in a container -- the *same* container that will be used in appsre and prow (the latter being the bigger challenge). Because even if we set `GOOS=linux` here, I don't think we can rely on the rest of the environment being same-enough to trust the resulting binary.

But in the meantime, I think this would be a reasonable thing to propose upstream.

Reply:
Yeah, I had to do that but I found it awful enough to avoid suggesting it in upstream 😂  But if you say so, we will

Reply:
Yeah, as long as we have some good comments around it, including acknowledging that it's temporary until we figure out how to containerize local builds, it'll be fine.